app/models/package_manager/sublime.rb

```diff
@@ -16,7 +16,9 @@ def self.project_names
     end
 
     def self.project(name)
-      get("https://packagecontrol.io/packages/#{CGI.escape(name)}.json")
+      Faraday.new("https://packagecontrol.io/packages/#{URI.escape(name)}.json")
+        .get
+        .then { |resp| resp.status == 200 ? Oj.load(resp.body) : nil }
```

Comment:
Our builtin `get` request method was following redirects, so it would end up on an HTML response instead of JSON and break.